sdcm/utils/features.py

```diff
@@ -0,0 +1,78 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+import logging
+
+from cassandra.cluster import Session  # pylint: disable=no-name-in-module
+
+CONSISTENT_TOPOLOGY_CHANGES_FEATURE = "SUPPORTS_CONSISTENT_TOPOLOGY_CHANGES"
+CONSISTENT_CLUSTER_MANAGEMENT_FEATURE = "SUPPORTS_RAFT_CLUSTER_MANAGEMENT"
+TABLETS_FEATURE = "TABLETS"
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_supported_features(session: Session) -> list[str]:
+    """
+    helper function to get supported_features from a running cluster,
+    if you need from a specific node use `patient_exclusive_cql_connection` session
+    """
+    result = session.execute("SELECT supported_features FROM system.local WHERE key='local'").one()
+    # NOTE: since row_factory can be different on different tests, we need to support multiple options
+    if isinstance(result, dict):
+        result = result["supported_features"].split(",")
+    elif isinstance(result, tuple):
+        result = result[0].split(",")
+    else:
+        raise NotImplementedError(f"unsupported row_factory={session.row_factory}")
+    LOGGER.debug("Supported features %s", result)
+    return result
+
+
+def get_enabled_features(session: Session) -> list[str]:
+    """
+    helper function to get supported_features from a running cluster,
+    if you need from a specific node use `patient_exclusive_cql_connection` session
+    """
+    result = session.execute("SELECT value FROM system.scylla_local WHERE key='enabled_features'").one()
+    # NOTE: since row_factory can be different on different tests, we need to support multiple options
+    if isinstance(result, dict):
+        result = result["value"].split(",")
+    elif isinstance(result, tuple):
+        result = result[0].split(",")
+    else:
+        raise NotImplementedError(f"unsupported row_factory={session.row_factory}")
+    LOGGER.debug("Enabled features %s", result)
+    return result
+
+
+def is_raft_schema_changes_feature_enabled(session: Session) -> bool:
```

Comment:
Func name is not name-compatible with the global var "CONSISTENT_CLUSTER_MANAGEMENT_FEATURE".
Is it intended?

Reply:
fixed to be consistent